### PR TITLE
Feature/complete UI

### PR DESCRIPTION
**Pull request by nammon72**
Complete UI for Calculator

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+document.getElementById("backspace").addEventListener("click", () => {
+  const lastElement = inputs.at(-1);
+
+  if (lastElement !== undefined) {
+    if (OPERATORS.includes(lastElement)) {
+      inputs.pop();
+    } else {
+      let lastElementStr = `${lastElement}`;
+      console.log(lastElementStr);
```
Comment (anhdiepmmk) [inline main.js]: remove this. DO NOT ADD console.log and commit this

File: main.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+btnDot.addEventListener("click", (event) => {
+  const lastElement = inputs.at(-1);
+
+  if (lastElement === undefined || OPERATORS.includes(lastElement)) {
+    // insert new element
+    inputs.push(event.target.textContent);
+    console.log("inputs-1: ", inputs);
+  } else if (typeof lastElement === "number" && !inputs.includes(".")) {
+    inputs.push(event.target.textContent);
+    console.log("inputs-2: ", inputs);
```
Comment (anhdiepmmk) [inline main.js]: remove

File: main.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    } else {
+      // merge with last element
+      inputs[inputs.length - 1] = parseFloat(
+        `${lastElement}${event.target.textContent}`
+      );
+      console.log(`inputs-2: ${inputs[inputs.length - 1]}`);
+    }
+
+    divResult.innerText = convertElements2String(inputs);
+    console.log(divResult.innerText);
```
Comment (anhdiepmmk) [inline main.js]: remove

File: main.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  });
+}
+
+btnDot.addEventListener("click", (event) => {
+  const lastElement = inputs.at(-1);
+
+  if (lastElement === undefined || OPERATORS.includes(lastElement)) {
+    // insert new element
+    inputs.push(event.target.textContent);
+    console.log("inputs-1: ", inputs);
```
Comment (anhdiepmmk) [inline main.js]: remove

File: .gitignore [hunk 5]
```diff
@@ -1,122 +1,5 @@
...
-.yarn/unplugged
-.yarn/build-state.yml
-.yarn/install-state.gz
-.pnp.*
 package.json
 package-lock.json
 node_modules
-.husky
 .idea
+*.test.js
```
Comment (anhdiepmmk) [inline .gitignore:5]: remove this

File: main.js (hunk 1 above)
Comment (nammon72) [reply main.js]: I will remove this at the branch: [https://github.com/anhdiepmmk/calculator/blob/feature/complete-UI/main.js](url)
Please check back

**Review by anhdiepmmk — APPROVED** [review]

**Pull request merged**